### PR TITLE
Docs: add alternative build method for mixins

### PR DESCRIPTION
**Pull request by wilfriedroset**
#### What this PR does

Add documentation about how to build mixin inside a container.

#### Checklist

- [ ] Tests updated
- [x] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by pracucci — APPROVED** [review]
Thanks! @osg-grafana could you review too, please?

File: docs/sources/operators-guide/visualizing-metrics/installing-dashboards-and-alerts.md [hunk 1]
```diff
@@ -34,12 +34,12 @@ If you choose this option, you can change the configuration to match your deploy
    ```bash
    git clone https://github.com/grafana/mimir.git
    ```
-2. Install dependencies
+2. Review the mixin configuration at `operations/mimir-mixin/config.libsonnet` and apply your changes if required
```
Comment (osg-grafana) [inline docs/sources/operators-guide/visualizing-metrics/installing-dashboards-and-alerts.md:37]:
```suggestion
2. Review the mixin configuration at `operations/mimir-mixin/config.libsonnet`, and apply your changes if necessary.
```

File: docs/sources/operators-guide/visualizing-metrics/installing-dashboards-and-alerts.md [hunk 2]
```diff
@@ -34,12 +34,12 @@ If you choose this option, you can change the configuration to match your deploy
...
+3. Install dependencies
```
Comment (osg-grafana) [inline docs/sources/operators-guide/visualizing-metrics/installing-dashboards-and-alerts.md:38]:
```suggestion
3. Install dependencies:
```

File: docs/sources/operators-guide/visualizing-metrics/installing-dashboards-and-alerts.md [hunk 3]
```diff
@@ -48,6 +48,11 @@ If you choose this option, you can change the configuration to match your deploy
 6. Install the recording rules saved at `operations/mimir-mixin-compiled/rules.yaml` in your Prometheus
 7. Install the alerts saved at `operations/mimir-mixin-compiled/alerts.yaml` in your Prometheus
 
+Alternatively, the mixin can be compile in a container. To do so, replace the previous steps 3) and 4) by the following command:
+  ```bash
+  make BUILD_IN_CONTAINER=true build-mixin
+  ```
```
Comment (osg-grafana) [inline docs/sources/operators-guide/visualizing-metrics/installing-dashboards-and-alerts.md:54]:
```suggestion
Alternatively, you can compile the mixin in a container. To do so, replace the previous two steps with the following command:
  ```bash
  make BUILD_IN_CONTAINER=true build-mixin
  ```
```

**Review by osg-grafana — APPROVED** [review]
Unblocking with feedback

File: docs/sources/operators-guide/visualizing-metrics/installing-dashboards-and-alerts.md (hunk 1 above)
Comment (osg-grafana) [reply docs/sources/operators-guide/visualizing-metrics/installing-dashboards-and-alerts.md:37]: This is a preference to match voice and tone, and to reduce the complexity of past participles.

File: docs/sources/operators-guide/visualizing-metrics/installing-dashboards-and-alerts.md (hunk 3 above)
Comment (osg-grafana) [reply docs/sources/operators-guide/visualizing-metrics/installing-dashboards-and-alerts.md:54]: Prefer active over passive voice.

**Comment by wilfriedroset** [comment]
Thank you for the review 🙏🏾
I reckon [errata-ai/vale](https://github.com/errata-ai/vale) would be helpful for the documentation PRs.
It seems like the tool is already used in at least one Grafana repository: https://github.com/grafana/k6-docs/tree/main/.vale

**Pull request merged**